### PR TITLE
Removed old ncl utilities.  Changed the random pertlim generator to a…

### PR DESCRIPTION
**Pull request by allibco**
Removed old ncl utilities.  Changed the random pertlim generator to allow more (and flexible) number of ensemble members.  Added the capability to generate 9 time step ensembles for UF-CAM-ECT.

Test suite: 
Test baseline: 
Test namelist changes: 
Test status: bit for bit

Fixes: compatible with pyCECT v3.0
User interface changes?:  yes

Code review:

### CODE REVIEW

**Comment by jedwards4b** [comment]
First please reedit the initial description and fill in the template, removing and replacing text in [].

File: tools/statistical_ensemble_test/single_run.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # $Id$
 # $URL$
 #
-# Sets up a 12-month ne30_ne30 run
+# Sets up a 12-month or ninth time step ne30_ne30 run
```
Comment (allibco) [inline tools/statistical_ensemble_test/single_run.sh]: thanks - i'll fix

File: tools/statistical_ensemble_test/single_run.sh [hunk 2]
```diff
@@ -293,7 +308,7 @@ case $MACH in
   ;;
 esac
 
-NewCaseFlags="$NewCaseFlags -res $RES -compset $COMPSET"
+NewCaseFlags="$NewCaseFlags -res $RES -compset $COMPSET --run-unsupported --project $ACCOUNT"
```
Comment (allibco) [inline tools/statistical_ensemble_test/single_run.sh]: Do you want me to make it so all must be -- ? Can you clarify :)

Comment (gold2718) [inline tools/statistical_ensemble_test/single_run.sh]: Double-dashes are required in most cases so fix as many as possible and note here any that cannot be fixed. See PR #1393.

Comment (allibco) [inline tools/statistical_ensemble_test/single_run.sh]: ok thanks - i'll try to fix

**Comment by allibco** [comment]
OK, I believe I've made the requested changes - and I updated the README file accordingly

**Pull request merged**